web/src/app/header/titlebar.component.html

```diff
@@ -15,11 +15,11 @@
 -->
 
 <h1 class="title">Kubernetes History Inspector</h1>
-<p class="version">v {{ ("NG_APP_VERSION" | env) || "Dev" }}</p>
-<p class="viewer-mode" *ngIf="'NG_APP_VIEWER_MODE' | env">
+<p class="version">{{ version }}</p>
+<!-- <p class="viewer-mode" *ngIf="'NG_APP_VIEWER_MODE' | env">
```

Comment:
Is it intended to keep this?

Reply:
I created another issue #46.
I need to add the API to get the viewer mode status from it instead of env variable on the build time.